.github/actions/repo-gardening/src/tasks/check-description/index.js

```diff
@@ -0,0 +1,305 @@
+	comment += `
+- ${
+		! body.includes( 'Testing instructions' ) ? `:red_circle:` : `:white_check_mark:`
+	} Add testing instructions.<br>`;
+
+	// Check for a proposed changelog entry.
+	comment += `
+- ${
+		! body.includes( 'Proposed changelog entry' ) ? `:red_circle:` : `:white_check_mark:`
+	} Include a changelog entry for any meaningful change.<br>`;
+
+	// Check if the Privacy section is filled in.
+	comment += `
+- ${
+		! body.includes( 'data or activity we track or use' ) ? `:red_circle:` : `:white_check_mark:`
+	} Specify whether this PR includes any changes to data or privacy.<br>`;
+
+	debug( `check-description: privacy checked. our comment so far is ${ comment }` );
+
+	comment += `
+
+
+This comment will be updated as you work on your PR and make changes. If you think that some of those checks are not needed for your PR, please explain why you think so. Thanks for cooperation :robot:
+
+******
+
+If you are an automattician, once your PR is ready for review add the "[Status] Needs Team review" label and ask someone from your team review the code.
+Once you’ve done so, switch to the "[Status] Needs Review" label; someone from Jetpack Crew will then review this PR and merge it to be included in the next Jetpack release.`;
+
+	// Gather info about the next release for that plugin.
+	const milestoneInfo = await buildMilestoneInfo( octokit, owner, repo, number );
+	comment += milestoneInfo;
+
+	// Look for an existing check-description task comment.
+	const existingComment = await getCheckComment( octokit, owner, repo, number );
+
+	const ownerLogin = owner.login;
+
+	// If there is a comment already, update it.
+	if ( existingComment !== 0 ) {
+		debug( `check-description: update comment ID ${ existingComment } with our new remarks` );
+		await octokit.issues.updateComment( {
+			owner: ownerLogin,
+			repo,
+			comment_id: +existingComment,
+			body: comment,
+		} );
+	} else {
+		// If no comment was published before, publish one now.
+		debug( `check-description: Posting comment to PR #${ number }` );
+
+		await octokit.issues.createComment( {
+			owner: ownerLogin,
+			repo,
+			issue_number: +number,
+			body: comment,
+		} );
+	}
+
+	// If some of our checks are failing, remove any "Needs Review" labels and add an Needs Author Reply label.
+	if ( comment.includes( ':red_circle:' ) ) {
+		debug( `check-description: some of the checks are failing. Update labels accordingly.` );
+
+		const hasNeedsReview = await hasNeedsReviewLabel( octokit, owner, repo, number );
+		if ( hasNeedsReview ) {
+			debug( `check-description: remove existing Needs review label.` );
+			await octokit.issues.removeLabel( {
+				owner: ownerLogin,
+				repo,
+				issue_number: +number,
+				name: '[Status] Needs Review',
+			} );
+		}
+
+		debug( `check-description: add Needs Author Reply label.` );
+		await octokit.issues.addLabels( {
+			owner: ownerLogin,
+			repo,
+			issue_number: +number,
+			labels: [ '[Status] Needs Author Reply' ],
+		} );
```

Comment:
Heh. If the only thing missing was hasStatusLabels, this just fixed it ;)

Reply:
That's a good point. As discussed in p1613665774177300-slack-CBG1CP4EN I've now made the status label check a bit more complex so it is not satified with a "Needs Author Reply" label. cbc8e42a362cda71f21f5710e27fb78ba8bf12a9